.github/workflows/dependabot.autoapprove.yml

```diff
@@ -0,0 +1,14 @@
+# Copyright 2023-2023 VMware, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+name: Auto approve
+on: pull_request_target
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
+    if: github.actor == 'dependabot[bot]'
```

Comment:
Should we also not filter the automerge action like this?

Reply:
May you elaborate?